src/realm/group.hpp

```diff
@@ -903,6 +913,15 @@ void Group::to_json(S& out, size_t link_depth, std::map<std::string, std::string
     out << "}";
 }
 
+inline std::size_t Group::compute_aggregated_byte_size() const noexcept
```

Comment:
Why is this function inlined? I think only functions that are performance critical should be inlined. Otherwise the header file gets polluted.